core/services/request/sanitizers/MultipleAttributes.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace EventEspresso\core\services\request\sanitizers;
+
+/**
+ * Class MultipleAttributes
+ *
+ * @author  Hossein Rafiei
+ * @package EventEspresso\core\services\request\sanitizers
+ * @since   4.10.30
+ */
+class MultipleAttributes
+{
+    /**
+     * @param string $attributes
+     * @param array $allowed_tags
+     * @param string $tag
+     * @return mixed|string
+     */
+    public static function clean(string $attributes, array $allowed_tags, string $tag = 'div')
```

Comment:
is there any reason you added the `$tag` parameter?
seems it would make zero difference what the tag was since we remove it anyways.
just hardcoding that to `div` would also mean we don't have to calculate the `strpos()` for the start.

Reply:
OHHHHHHH   yes of course!

great awesome, thnx for the explanation!

Great job then and plz disregard my comments about changing that 😆